src/index.html

```diff
@@ -7,7 +8,57 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  <!-- #endregion -->
+
   <body>
-    <h1>Stars</h1>
+    <div>
+      <div class="stars stars--0">
+        <div class="stars__star stars__star--first"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+      </div>
+
+      <div class="stars stars--1">
+        <div class="stars__star stars__star--first stars__star--active"></div>
```

Comment:
```suggestion
        <div class="stars__star"></div>
```
redundant classes. to style first element use :first-child 
for amount of active stars will be responsible stars--1 modifier and :nth-child pseudo class
remove for all blocks